src/lib/features/feature-toggle/tests/feature-toggle-service.limit.test.ts

```diff
@@ -91,6 +91,44 @@ describe('Strategy limits', () => {
         );
     });
 
+    test('Should not throw limit exceeded errors if the new number of constraints is less than or equal to the previous number', () => {
```

Comment:
do we also have a test for should throw?

Reply:
We already have a number of tests in place that checks that it throws when you reach the limit, right? But I can add another one in if you think that's useful?

Reply:
Like I can add a little:

```ts
        expect(() => {
            featureToggleService.validateConstraintsLimit({
                updated: [...constraints, ...constraints],
                existing: constraints,
            });
        }).toThrow(ExceedsLimitError);
```

To the end, for instance?

Reply:
Also added in 17ee7675bfa50707bbb8b14f363f77ba00d8cb91